scss/_custom-styles.scss

```diff
@@ -246,3 +246,9 @@
     column-count: 4;
   }
 }
+
+// >> Border thick utility.
+
+.border-thick {
+  border-width: .25rem;
```

Comment:
Noticing that the docs say there's an `!important` here, but it is missing. Adding it on a local build produces nice 
thick borders on the example.

Reply:
The linting checks didn't like when I had the `!important` here at first. Did your local build remove the `!important` from  `.border`? If that's removed, then this `!important` isn't needed (and other added benefits).

Reply:
Verified that the local linting checks didn't like adding `!important`: looking next at the order in which the rules are applied (I didn't change anything else).